ldk/javascript/src/whisper/whisper-mapper.ts

```diff
@@ -0,0 +1,34 @@
+import { NewWhisper, UpdateWhisper } from ".";
+import { whisper } from "..";
+import { parseForm } from "./form-parser";
+
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */ // We have to coerce any type here to force conversion of whisper.NewWhisper to OliveHelps.NewWhisper 
+export function convertComponentType(whisperComponentType: whisper.WhisperComponentType): any {
```

Comment:
Not sure if this is really a mapper or conversion function. Looks more like a validation. It is strange that we will throw an error here. Perhaps we could introduce a validation layer?